firmware/user/modes/mode_flight.c

```diff
@@ -710,17 +710,29 @@ static bool ICACHE_FLASH_ATTR flightRender()
 		int x,y;
 		int overlay = 0;
 		//1,000 triangles @ 28.3ms.
-		ij = 32;
-		for( overlay = 0; overlay < 10; overlay++ )
-		for( y = 0; y < 10; y++ )
-		for( x = 0; x < 10; x++ )
+		if( tflight->perfMotion )
+		{
+			for( overlay = 0; overlay < 100; overlay++ )
+			{
+				int col = os_random()%2;
+				outlineTriangle( (os_random()%256)-64, (os_random()%128)-32, (os_random()%256)-64, (os_random()%128)-32,
+					(os_random()%256)-64, (os_random()%128)-32, col, !col );
+			}
+		}
+		else
 		{
-			int mx = x * 12;
-			int my = y * 6;
-			int mx1 = x*12+tdSIN( ij+x+y )/25;
-			int my1 = y*6+tdCOS( ij+x+y )/25;
-			outlineTriangle( mx, my, mx1, my, mx, my1, 0, 1 );
-			outlineTriangle( mx, my1, mx1, my1, mx1, my, 0, 1 );
+			ij = 32;
+			for( overlay = 0; overlay < 10; overlay++ )
+			for( y = 0; y < 10; y++ )
+			for( x = 0; x < 10; x++ )
+			{
+				int mx = x * 12;
+				int my = y * 6;
+				int mx1 = x*12+tdSIN( ij+x+y )/25;
+				int my1 = y*6+tdCOS( ij+x+y )/25;
+				outlineTriangle( mx, my, mx1, my, mx, my1, 0, 1 );
+				outlineTriangle( mx, my1, mx1, my1, mx1, my, 0, 1 );
+			}
 		}
 #ifndef EMU
         OVERCLOCK_SECTION_DISABLE();
```

Comment:
Need to add  clearDisplay(); to normal game (actually about 10 lines down from here)